ios/MaasCore/Package.swift

```diff
@@ -0,0 +1,45 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MaasCore",
+    platforms: [
+        .iOS(.v13),
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "MaasCore",
+            targets: ["MaasCore", "MaasComponents", "MaasTheme"]),
+    ],
+    dependencies: [
+        .package(
+            name: "Swappable",
+            url: "https://github.com/trafi/swappable.git", .upToNextMinor(from: "0.0.1")),
```

Comment:
Will each `Package.swift` in our project need to define this same dependency and version? Can we centralize it in one place? Is it a hassle to update these versions if we use fixed versions?

Reply:
This required when we want to develop not on whole **Maas** package but directly on **MaasCore** package.